### DISCUSSION TITLE
ReferenceError: Property 'viewManagersMetadata' doesn't exist, js engine: hermes

### DISCUSSION THREAD

**Discussion by Makeit-Tipi** [expo/expo#27847] [Archived: Questions] [answered]
After upgrading from SDK 48 to SDK 50, both of the Expo packages I was using caused my app to crash when running:
`npx expo start --no-dev --minify`

In the device log, I only see the following error: `ReferenceError: Property 'viewManagersMetadata' doesn't exist, js engine: hermes`

When I stop using these two packages, **expo-blur** and **expo-linear-gradient**, the app no longer crashes.

When building a fresh project and adding theses packages, I'm not able to reproduce it.

Do you have any idea how to fix this with these two packages?

Here are all the packages used in my project

<img width="544" alt="Screenshot 2024-03-25 at 15 42 57" src="https://github.com/expo/expo/assets/88028094/3f466583-16e2-4d0c-b7d2-01e503b7463d">
<img width="525" alt="Screenshot 2024-03-25 at 15 43 06" src="https://github.com/expo/expo/assets/88028094/b9506644-18d2-4428-b882-a303e1881a15">

Thanks 🙏

**Comment by tsapeta** [comment]
Does it work in debug, i.e. without `--no-dev --minify`?

**Reply by Makeit-Tipi** [reply]
Yes in debug mode the app is running without any problem

**Comment by Makeit-Tipi** [comment, marked as answer]
but I managed to fix it by removing the `react-native-reanimated/plugin` from my **babel.config.js**

<img width="505" alt="Screenshot 2024-03-27 at 10 43 04" src="https://github.com/expo/expo/assets/88028094/435236da-8efb-437c-aa53-f6ea6a82ff5f">

 Hope it will help someone

**Reply by tsapeta** [reply]
That's interesting, I wouldn't expect such errors caused by the plugin! 🤔 However, `babel-preset-expo` already includes the Reanimated plugin, so probably it ran twice.
